garak/_config.py

```diff
@@ -23,7 +23,9 @@
 
 DICT_CONFIG_AFTER_LOAD = False
 
-version = -1  # eh why this is here? hm. who references it
+from garak import __version__
+
+version = __version__
```

Comment:
`cli` actually overrides this on startup, if moving the lookup here we should remove there:
https://github.com/leondz/garak/blob/71d47aa67ecde3afa784861468fced03d65000a9/garak/cli.py#L19-L34

Reply:
Thanks! Saw this, went for a more generic solution, forgot to prune the activity in `cli`